server/api/controllers/todolist.controller.js

```diff
@@ -0,0 +1,53 @@
+const asyncMiddleware = require('../middlewares/asyncMiddleware');
+
+module.exports = function TodolistController(gladys) {
+  /**
+   * @api {get} /api/v1/todolist/tasks Get Todolist tasks
+   * @apiName getTasks
+   * @apiGroup Todolist
+   */
+  async function getTasks(req, res) {
+    const tasks = await gladys.todolist.getTasks();
+    res.json(tasks);
+  }
+
+  /**
+   * @api {get} /api/v1/todolist Get Todolist
```

Comment:
A question, what does `Get Todolist` means?

Reply:
I have change the comment to `Get list of todolist`